client/my-sites/plan-features/style.scss

```diff
@@ -195,6 +195,10 @@ $plan-features-header-banner-height: 20px;
 		border-bottom: none;
 	}
 
+	&:first-child {
+		border-top: solid 1px lighten( $gray, 27% );
```

Comment:
Notice the border is wider in @apeatling's design:

![selection_034](https://cloud.githubusercontent.com/assets/4988512/16522548/d99a8144-3f9e-11e6-84ee-270f0e9b402b.jpg)

I think it's nicer with wider border because this border separates the "PlanFeaturesFooter" and PlanFeaturesItemList.